src/pyodide/internal/python.ts

```diff
@@ -51,6 +51,8 @@ function prepareWasmLinearMemory(Module: Module): void {
     // The effects of these are purely in Python state so they only need to be run
     // if we didn't restore a snapshot.
     entropyBeforeTopLevel(Module);
+    // Note that maybeAddVendorDirectoryToPath runs after adjustSysPath and rearranges where
+    // the /session/metadata path is added.
     adjustSysPath(Module);
```

Comment:
Should we only call this here in version 0.26.0a2 and in other cases leave all the path changes to (renamed) `maybeAddVendorDirectoryToPath()`?

Reply:
Okay, I've rearranged things a bit. Hoping that doesn't break it.